bin/runner.js

```diff
@@ -101,6 +102,10 @@ function generate(model, allTestsGenerated) {
   });
 }
 
+function cleanup(model) {
+  rimraf.sync(model.testsDocPath);
```

Comment:
I worry about doing this freely: what if '/home/noah/' ends up as the testsDocPath?

Reply:
totally and `Doc` is not that special.

Reply:
I've renamed the folder `Doc` to `VerifyExamples`. This partially addresses your concern and is also nice in error messages.

<img width="394" alt="fish___users_stoeffel_elm-verify-examples" src="https://user-images.githubusercontent.com/1217681/34312918-d2dbb0d6-e767-11e7-853d-a22c456156ee.png">

Reply:
👍